README.md

```diff
@@ -2,10 +2,10 @@
 
 The Warning Signs gem builds upon the ideas of the
 [Ruby Warning Gem](https://github.com/jeremyevans/ruby-warning) 
-and [Stop Ignoring Your Ruby and Rails Deprecations](https://blog.testdouble.com/posts/2023-04-24-stop-ignoring-your-ruby-and-rails-deprecations/).
+and [Stop ignoring your Rails (and Ruby) deprecations!](https://blog.testdouble.com/posts/2023-04-24-stop-ignoring-your-ruby-and-rails-deprecations/).
 
 The idea is to provide an easily-configurable way to manage deprecation 
-warnings from Ruby and Rails to support your upgrade tool.
+warnings from Ruby and Rails to support your upgrades.
```

Comment:
"upgrade tool" didn't feel like the right phrase here, but I'm not sure "upgrades" is either. What are your trying to communicate here?